galaxy_importer/utils/execution_environment.py

```diff
@@ -0,0 +1,116 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import os
+import re
+
+from galaxy_importer.utils import yaml as yaml_utils
+
+
+def process_execution_environment(path, logger):
+    ex_env = {}
+    ex_env_path = os.path.join(path, 'meta', 'execution-environment.yml')
+
+    if os.path.exists(ex_env_path):
+        ex_env = _load_yaml(ex_env_path, logger)
+    else:
+        logger.info('No execution environment data found.')
+        return ex_env
```

Comment:
Here this is an empty dict `{}`, to get parity with what `ansible-builder` is doing, you could return something like:

```json
{
  "dependencies": {
    "python": "requirements.txt",
    "system": "bindep.txt"
  }
}
```

...conditionally depending on whether each of those 2 files existed.